app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationListItem.tsx

```
@@ -33,8 +36,85 @@ export const ConversationListItem = ({
+  const { category } = useParams<{ category: string }>();
   const { searchParams } = useConversationsListInput();
   const searchTerms = searchParams.search ? searchParams.search.split(/\s+/).filter(Boolean) : [];
+  const { toast } = useToast();
+  const utils = api.useUtils();
+
+  const { mutate: updateStatus, isPending: isUpdating } = api.mailbox.conversations.update.useMutation();
+
+  const handleUpdateStatus = (e: React.MouseEvent, newStatus: "open" | "closed" | "spam") => {
+    e.stopPropagation();
+    const previousStatus = conversation.status;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)